src/chatbot.html

```diff
@@ -14,11 +14,15 @@
 <body>
 	<div class="container" >
 		<div class="top-bar" id="nav" >
-      <a href="https://susi.ai/">
-        <img class="logo" src="images/susi_logo_white.png"></a>
+      	<a href="https://susi.ai/">
+        	<img class="logo" src="images/susi_logo_white.png">
+        </a>
+        <img class="settings-icon" id="settingsIcon" src="images/settings.SVg">
```

Comment:
There is a small typo in this line that prevents the settings svg to load.
![typo](https://user-images.githubusercontent.com/20047982/31288522-e514d56e-aae2-11e7-8d33-f8d970de2b73.png)
Should I open an issue?
This could be a good first issue.

Reply:
It is still rendering for me. But I guess it should be named properly. Please go ahead and create an issue
![screen shot 2017-10-06 at 10 35 46 pm](https://user-images.githubusercontent.com/11137394/31289550-b80e817e-aae6-11e7-8c84-03e0b4dbb8ff.png)